javaslang/src/main/java/javaslang/collection/Vector.java

```diff
@@ -11,9 +11,12 @@
 
 import java.io.Serializable;
 import java.util.*;
+import java.util.concurrent.atomic.AtomicReference;
```

Comment:
K, I will introduce a Javaslang `MutableWrapper` for this then, so that we can assign values from inside a lambda.
@danieldietrich?